Formula/percona-server.rb

```diff
@@ -72,13 +72,8 @@ def install
       -DCOMPILATION_COMMENT=Homebrew
       -DWITH_EDITLINE=system
       -DCMAKE_BUILD_TYPE=RelWithDebInfo
-    ]
-
-    # PAM plugin is Linux-only at the moment
```

Comment:
The purpose of this PR is to add the `dialog.so` plugin so the `mysql` command line and other programs like `Sequel Pro` and `MySQLWorkbench` can use it.

```
MySQL said: Authentication plugin 'dialog' cannot be loaded: dlopen(/usr/local/lib/plugin/dialog.so, 2): image not found
```

The Percona PAM authentication plugins for MySQL are not built unless `WITH_PAM` is set.